Potatso/AppInitializer.swift

```diff
@@ -36,15 +36,15 @@ class AppInitializer: NSObject, AppLifeCycleProtocol {
         fileLogger?.logFileManager.maximumNumberOfLogFiles = 7
         DDLog.add(fileLogger)
 
-        let logglyLogger = LogglyLogger() // Loggy Logger
-        logglyLogger.logglyKey = InfoInternal.shared.getLogglyAPIKey()
-        let fields = LogglyFields()
-        fields.userid = User.currentUser.id
-        fields.appversion = AppEnv.fullVersion
-        let formatter = LogglyFormatter(logglyFieldsDelegate: fields)
-        formatter?.alwaysIncludeRawMessage = false
-        logglyLogger.logFormatter = formatter
-        DDLog.add(logglyLogger)
+//        let logglyLogger = LogglyLogger() // Loggy Logger
+//        logglyLogger.logglyKey = "107d98a8-c275-4369-a881-b36d3564c9ef"
```

Comment:
What happens here. I think this PR should not involve touching too many thing. LogglyLogger can be left intact, and code before changed grabs api key from `Info.plist` nicely, no need to hardcoded.

Reply:
I don't know to use LogglyLogger. I had register an account in "loggly.com". I use customer token replace the logglyKey. But it had a connection error when I run Potatso. So I remove it.

Reply:
Today, I try again, It seems normally.
Suspending, posting logs to Loggly
2018-01-10 11:35:15.647177+0800 Potatso[1576:769317] Posting to Loggly: {"rawlogmessage":

Reply:
Is it working before before you made change? `InfoInternal` will grab API key from `Info.plist` automatically.

Reply:
Anyway, **in future** I think we can remove it as developer will build their own app and change API key anyway.